.rubocop.yml

```diff
@@ -4,3 +4,9 @@ LineLength:
   Max: 99
 Style/FrozenStringLiteralComment:
   Enabled: false
+Metrics/ModuleLength:
+  Exclude:
+    - "**/*_spec.rb"
```

Comment:
exclude from all cops the rails_helper and spec_helper, but we want to run rubocop in the specs